content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -133,7 +133,7 @@ iptables:
 
 ##### Performance optimization for `iptables` mode {#minimize-iptables-restore}
 
-{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+{{< feature-state for_k8s_version="v1.28" state="stable" >}}
 
 In Kubernetes {{< skew currentVersion >}} the kube-proxy defaults to a minimal approach
```

Comment:
This might well need a rewrite: I'm assuming that the feature gate is going to be GA and locked to `true`, so “_defaults to_” isn't correct any more.